azure-pipelines-official.yml

```diff
@@ -308,14 +308,6 @@ stages:
     pool:
       vmImage: windows-2019
     steps:
-    - checkout: none
-
-    - task: NuGetCommand@2
-      displayName: 'Install RIT from Azure Artifacts'
-      inputs:
-        command: custom
-        arguments: 'install RoslynTools.VisualStudioInsertionTool -PreRelease -Source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json'
-
     - powershell: |
```

Comment:
should this script be in the shared insert.yml?

Reply:
@JoeRobich I don't think so - this sets variables like draft PR creation, title prefix, toolset insertion, VS branch to insert into, etc based on variables in PublishData.json.  

However in PR validation the branches are usually not in PublishData.json.  The draft PR / prefix are fixed for the PR builds, and the majority of the time we are inserting into VS main for PR val.  When that is not true, I have defined pipeline variables in the PR val pipeline to specify exactly which branch should be used.  This is essentially mirroring what the current release pipeline does as well.

Reply:
Personally I think it would be beneficial to keep the bit for auto setting the VS Branch, since typically we run the PR Validation against the release branch we expect to merge into. I think it would be fine to fall back to VS main when the branch isn't a release branch.